File: frontend/src/app/models/transferencia.ts
export class Transferencia
{
    importe: number;
    detalles: string;
    idCuentaOrigen: number;
    idCuentaDestino: number;

    constructor(importe: number, detalles: string, idCuentaOrigen: number, idCuentaDestino: number) {
        this.importe = importe;
        this.detalles = detalles;
        this.idCuentaOrigen = idCuentaOrigen;
        this.idCuentaDestino = idCuentaDestino;
    }
}

File: backend/WalletAPI/Entities/Transferencia.cs
using System;
using System.Collections.Generic;

namespace Entities
{
    public partial class Transferencia
    {
        public int? IdTransferencia { get; set; }
        public decimal Importe { get; set; }
        public string? Detalles { get; set; }
        public DateTime FechaHora { get; set; }
        public int IdCuentaOrigen { get; set; }
        public int IdCuentaDestino { get; set; }

        public virtual Cuenta? IdCuentaDestinoNavigation { get; set; } = null!;
        public virtual Cuenta? IdCuentaOrigenNavigation { get; set; } = null!;
    }
}
